exercises/triangle/.meta/template.j2

```diff
@@ -0,0 +1,12 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header(["equilateral", "isosceles", "scalene"]) }}
+
+{% for case in cases -%}
+class {{ case["description"] | camel_case }}Test(unittest.TestCase):
+    {% for case in case["cases"] %}
+    def test_{{ case["description"] | to_snake }}(self):
+        self.assertEqual({{ case["property"] }}({{ case["input"]["sides"] }}), {{ case["expected"] }})
```

Comment:
OK, smaller diff let me see this one. From our [conventions](https://github.com/exercism/python#conventions):
> We use assertIs(actual, True) and assertIs(actual, False) rather than assertTrue(actual) or assertFalse(actual) (#419).

```suggestion
        self.assertIs({{ case["property"] }}({{ case["input"]["sides"] }}), {{ case["expected"] }})
```

Reply:
changed to assertIs as per conventions. Thanks.